<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSerialConfig</class>
 <widget class="QWidget" name="frmSerialConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>指令结尾</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  帧结尾</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pbOpenPort">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开 始 监 听</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="pbClosedPort">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关 闭 接 收</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>   操作</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="text">
      <string>0x88</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="text">
      <string>0xde</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>0xda</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  帧长度</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="text">
      <string>0x85</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>间隔字节</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pbCreateServer">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>创建服务器</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>    指令</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>服务器IP</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前状态</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>指令起始</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  帧起始</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="text">
      <string>0xfe</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="IP_address">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>端 口 号</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="Portname"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="ServerState">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Closed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pbSendCommand">
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发 送 指 令</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>客户端连接</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="connectstate">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>no connection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
